check_vitals.py

```diff
@@ -0,0 +1,43 @@
+
+from time import sleep
+import sys
+
+class check_vitals:
+  def is_temperature_ok(self, temperature):
+    if temperature in range(95, 102):
+      self.print_wait_time()
```

Comment:
The purpose of printing and waiting is to grab the user's attention. But we don't need to do that when the temperature is normal.